third_party/nvfuser/csrc/python_frontend/fusion_record.h

```diff
@@ -1099,9 +1099,10 @@ struct TensorRecord : RecordFunctor {
   }
 
   virtual void operator()(FusionState& fd) final {
+    // TODO:fix this funciton with Jie's PR
     auto tv = TensorViewBuilder()
                   .ndims(symbolic_sizes_.size())
-                  .contiguity(contiguous_info_)
+                  // .contiguity(contiguous_info_)
```

Comment:
This is only temporary, will be replaced by https://github.com/csarofeen/pytorch/pull/2561